### PR TITLE
Enhance discount rate callout to actually work

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2348** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2348" target="_blank">Graphite</a> 
    * **PR #2349** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2349" target="_blank">Graphite</a> 
      * **PR #2350** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2350" target="_blank">Graphite</a> 
        * **PR #2351** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2351" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2351?utm_source=stack-comment).

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2351?utm_source=stack-comment).


Job Reference: FCO7UxXsqYOBvxApXFkP

File: src/components/Create/components/pages/ProjectToken/components/CustomTokenSettings/CustomTokenSettings.tsx [hunk 1]
```diff
@@ -75,32 +112,69 @@ export const CustomTokenSettings = () => {
...
+                <>
+                  <Trans>
+                    Contributors will receive {discountRate}% more tokens for
+                    contributions they make this funding cycle compared to the
+                    next funding cycle.
+                  </Trans>
+                  <Trans>
+                    The issuance rate of your second funding cycle will be{' '}
+                    {formatAmount(secondFundingCycleMintRate)} tokens per 1 ETH,
+                    then {formatAmount(thirdFundingCycleMintRate)} tokens per 1
```
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectToken/components/CustomTokenSettings/CustomTokenSettings.tsx:170]:
```suggestion
                     {formatAmount(thirdFundingCycleMintRate)} tokens per 1
```

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2351?utm_source=stack-comment).


Job Reference: tq0DBErRQ758wBw1xKNP

**Pull request merged**